web/rest/platform/config/api/raw/provider.yml

```diff
@@ -103,14 +103,18 @@ Ivoz\Provider\Domain\Model\Administrator\Administrator:
         raw: "id > 0 or id === null"
 
 Ivoz\Provider\Domain\Model\Currency\Currency:
-  itemOperations: []
+  itemOperations:
+    get: ~
+    put: ~ 
+    delete: ~
   collectionOperations:
     get: ~
+    post: ~
   attributes:
     access_control: '"ROLE_SUPER_ADMIN" in roles && user.hasAccessPrivileges(_api_resource_class, request.getMethod())'
     write_access_control:
       ROLE_SUPER_ADMIN:
-        raw: "FALSE"
+        raw: "id > 0 or id === null"
```

Comment:
unnecessary

Reply:
If don't set this parameter, it will revoke access to the new endpoints.